docs/source/using/troubleshooting.adoc

```diff
@@ -113,6 +113,15 @@ cause passwords to fail.
 Workaround: When creating and entering passwords, wrap the string with
 single quotes in the following format: `<password>`
 
+NOTE: The characters `", $, ` and \` are still interpreted by the shell,
```

Comment:
> I agree and I really want some good wording around it why we need it.

Ok, as long as we agree on this. However, it will always be hard to explain, unless we don't fix the underlying problem. For the user, using the single quotes on the console is all which should be required. He does the right thing. 

So ssh does not use the single quote syntax to escape the password and it also does not handle spaces. Fair enough. In this case we have to do whatever ssh expect and that is escaping these meta characters. Why offloading this to the user and creating confusion. We have the password string, right? All we need is a single function which takes the password and escapes it to ssh standards. Am I missing something here?